### DISCUSSION TITLE
Github Projects Webhook Is Not Working In Discord 🙀

### DISCUSSION THREAD

**Discussion by volkantash** [community/community#38951] [Projects and Issues]
### Select Topic Area

Bug

### Body

Hi,

Github projects webhook is not working in Discord. Yes, webhook's other info... Coming to Discord channel. But Discord can't read Projects steps in Discord channel.

**Comment by BillyDotWS** [comment]
Also noticed, seems like projects v2 isn't supported in discord yet.

I don't think this is a github issue, it's a discord issue, try a webhook outside of discord and see if it gives information.

**Reply by volkantash** [reply]
I do not think so.

**Comment by volkantash** [comment]
Months passed. Has anyone developed a solution? 🤔

**Reply by Firewolf1337** [reply]
https://gist.github.com/SGTGunner/50d6a3cc0d489cf779f77695ba3e22ea
here is the solution in the first reply.


Just add /github to the webhook link

**Reply by volkantash** [reply]
> https://gist.github.com/SGTGunner/50d6a3cc0d489cf779f77695ba3e22ea here is the solution in the first reply.
> 
> Just add /github to the webhook link

This is not answer for my question. Repeat read it please. Need repair 👉 **Github Projects 👈 Webhook** for Discord. Thanks.

**Comment by Deep0Thinking** [comment]
Facing the same issue here with **Github Projects** webhook and **Discord**. Also found that **label changes** notifications for **Github repo** Webhooks do not work in **Discord**. Any updates or fixes yet?

Update:

I tested, the **Github Projects** webhook one is the **Github Projects** side problem, and the **label changes** one is the **Discord** side problem.

**Comment by alifeee** [comment]
Hi.

If you use the webhook format <https://discord.com/api/webhooks/[webhook id]/[token]/github>, then it is likely that the event is not supported.

See <https://github.com/discord/discord-api-docs/issues/6203#issuecomment-1608151265> for a current list of supported events.

As of June 26, 2023, the list is (from the above link):

> - commit_comment
> - create
> - delete
> - fork
> - issue_comment
> - issues
> - member
> - public
> - pull_request
> - pull_request_review
> - pull_request_review_comment
> - push
> - release
> - watch
> - check_run
> - check_suite
> - discussion
> - discussion_comment
